cli/src/commands/asset.ts

```diff
@@ -201,9 +227,9 @@ export const uploadFiles = async (files: string[], { dryRun, concurrency }: Uplo
     { concurrency, retry: 3 },
   );
 
-  for (const filepath of files) {
-    await queue.push(filepath);
-  }
+  files.map((item) => {
+    void queue.push(item);
+  });
```

Comment:
Can you just use a for loop here please?

Reply:
Oops, remainder of a previous piece of code, sry